client/elements/text/sc-simple-text.html

```diff
@@ -405,6 +401,27 @@ <h3>Could not find the sutta text.</h3>
               return firstSection;
           }
 
+          _paddingToMargin(section) {
+            const margin = window.getComputedStyle(section).padding;
+            requestAnimationFrame(() => {
+                section.style.margin = margin;
+                section.style.padding = '0';
+            })
+          }
+
+          _marginToPadding(section) {
+            for(let pos of ['Top', 'Bottom', 'Left', "Right"]) {
```

Comment:
making consecutive reads and writes to `style` inside a `for` loop is really bad practice, as the browser needs to rebuild the layout before each read. This loop only runs 4 times, so it's borderline acceptable. 

It would still be much better if this was split into two loops. In the first one you'd read the margins/paddings into an object and then write to the DOM using the values already in that object.